code/modules/mob/living/carbon/carbon.dm

```diff
@@ -360,6 +360,7 @@
 	if(cuff_break)
 		. = !((I == handcuffed) || (I == legcuffed))
 		qdel(I)
+		SEND_SIGNAL(src, COMSIG_CARBON_CUFF_ESCAPE, src)
```

Comment:
We should generally only be sending signals in one place so that it's trivial to change their arguments and to add cancelling etc. Make this all a `cuff_escape()` proc or similar